aesara/graph/opt.py

```diff
@@ -1694,7 +1694,8 @@ def transform(self, fgraph, node, get_nodes=True):
                 if ret is not False and ret is not None:
                     assert len(real_node.outputs) == len(ret)
                     if self.values_eq_approx:
-                        ret.tag.values_eq_approx = self.values_eq_approx
+                        for r in ret:
```

Comment:
I guess this branch was never hit in the tests because it's obviously a bug, as `ret` is supposed to be an iterable given the `assert` above